idd/Energy+.idd.in

```diff
@@ -40267,9 +40273,9 @@ ZoneHVAC:TerminalUnit:VariableRefrigerantFlow,
         \note AirConditioner:VariableRefrigerantFlow:FluidTemperatureControl or
         \note AirConditioner:VariableRefrigerantFlow:FluidTemperatureControl:HR
         \note is used to model VRF outdoor unit
-        \default Fan:ConstantVolume
```

Comment:
This one is a little more tricky. If there's a fan name, but this field is blank, then the get input should look for a Fan:ConstantVolume to avoid breaking an old idf. Maybe you've already allowed for that.